src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -0,0 +1,157 @@
+                    {
+                        nameof(FingerprintPhraseFormatted),
+                        nameof(LogInAttemptByLabel),
+                        nameof(TimeOfRequestText),
+                    });
+            }
+        }
+
+        private FormattedString CreateFingerprintPhrase(string fingerprintPhrase)
+        {
+            try
+            {
+                return fingerprintPhrase?.Split(new[] { Constants.FingerprintPhraseSeparator }, StringSplitOptions.RemoveEmptyEntries).Aggregate(new FormattedString(), (fs, fingerprint) =>
+                {
+                    if (fs.Spans.Any())
+                    {
+                        fs.Spans.Add(new Span
+                        {
+                            Text = Constants.FingerprintPhraseSeparator,
+                            TextColor = ThemeManager.GetResourceColor("DangerColor")
+                        });
+                    }
+
+                    fs.Spans.Add(new Span
+                    {
+                        Text = fingerprint
+                    });
+
+                    return fs;
+                });
+            }
+            catch (Exception ex)
+            {
+                _logger.Exception(ex);
+                return null;
+            }
+        }
+
+        private string CreateRequestDate(DateTime requestDate)
+        {
+            var minutesSinceRequest = requestDate.ToUniversalTime().Minute - DateTime.UtcNow.Minute;
+            if (minutesSinceRequest < 5)
+            {
+                return AppResources.JustNow;
+            }
+            if (minutesSinceRequest < 59)
+            {
+                return string.Format(AppResources.XMinutesAgo, minutesSinceRequest);
+            }
+
+            return requestDate.ToShortTimeString();
+        }
+
+        private async Task AcceptRequestAsync()
+        {
+            await _deviceActionService.ShowLoadingAsync(AppResources.Loading);
+            var res = await _authService.LogInPasswordlessAcceptAsync();
+            await _deviceActionService.HideLoadingAsync();
+            await Page.Navigation.PopModalAsync();
+            _platformUtilsService.ShowToast("info", null, AppResources.LogInAccepted);
+        }
+
+        private async Task RejectRequestAsync()
+        {
+            await _deviceActionService.ShowLoadingAsync(AppResources.Loading);
+            var res = await _authService.LogInPasswordlessRejectAsync();
+            await _deviceActionService.HideLoadingAsync();
+            await Page.Navigation.PopModalAsync();
+            _platformUtilsService.ShowToast("info", null, AppResources.LogInDenied);
+        }
+
+        private void HandleException(Exception ex)
+        {
+            Xamarin.Essentials.MainThread.InvokeOnMainThreadAsync(async () =>
+            {
+                await _deviceActionService.HideLoadingAsync();
+                await _platformUtilsService.ShowDialogAsync(AppResources.GenericErrorMessage);
+            });
```

Comment:
Even though this won't crash, we need to log the exception if something happens inside. Right now if `_platformUtilsService.ShowDialogAsync` throws the exception, it will be swollen by the `Task`.
You can try adding `FireAndForget()` at the end and check if it works as expected (it should work but maybe I'm not considering something and it changes the thread execution).